### ISSUE TITLE
Doesn't work on MacOS 14 Sonoma

### ISSUE THREAD

**Issue by bog-dance** [lg/HowOnline#12]
After the update to MacOS 14.0 it stopped working and just showed "no ssid". 
I understand, you probably don't support this project anymore or even forgot about it:) But I've been using this indicator for 5 years, I'm traveling a lot, changing lots of wifi, and ping to google on my toolbar is an essential indicator of the quality of the network. Is it possible to fix it ?:)
<img width="712" alt="Screenshot 2023-10-28 at 09 10 59" src="https://github.com/lg/HowOnline/assets/15802974/82b06a88-c528-48d3-ad81-4f960d7c7dd6">

**Comment by jcfpwr** [comment]
Hi, same problem, I was wondering if the bug was between the table and the chair, but your message confirms that sonoma broke HowOnline ...

**Comment by jcfpwr** [comment]
I've just had the idea to erase it with AppCleaner and réinstalling, but it hasn't changed anything.

**Comment by VsevolodUstinov** [comment]
Same problem here. I love this product, and I hope will be fixed soon

**Comment by smorrel1** [comment]
+1
You built an amazing tool which I used for years on my Mac. Thank you!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
It has been working until now but 1.01(4) now fails on my Mac Sonoma 14.1.2.
Any similar tools out there?

**Comment by jcfpwr** [comment]
May be we should try to ask an AI to help us to fix the code ?

**Comment by jcfpwr** [comment]
I havent found any alternative to this so good util. I've tried to have a shortcut to a webpage doing ping in my menubar with Menubarx but without success yet...

**Comment by bog-dance** [comment]
> May be we should try to ask an AI to help us to fix the code ?

Yeah, agree. Let's feed some open-source model, like Llama, by all the parsed github and ask resulted dude, How online? It will be much easier, than open and dive into old Obj-C/Swift package.  

> I havent found any alternative to this so good util. I've tried to have a shortcut to a webpage doing ping in my menubar with Menubarx but without success yet...

But while we train models, I've found some [Ping Menu](https://github.com/kalleboo/PingMenu) tool. The last commit here is even older, than in our How Online. But it written without swift, on a 100% Obj-C and it works on my updated MacOS. 
The only thing, it shows latency in a bit annoying seconds ( 0.032s), but not in milliseconds.

**Comment by bog-dance** [comment]
The only thing, it shows latency in a bit annoying seconds ( 0.0032s), but not in milliseconds.
